lib/ansible/modules/network/nxos/nxos_portchannel.py

```diff
@@ -129,7 +129,6 @@
 from ansible.module_utils.nxos import get_config, load_config, run_commands
 from ansible.module_utils.nxos import nxos_argument_spec, check_args
 from ansible.module_utils.basic import AnsibleModule
-from ansible.module_utils.netcfg import CustomNetworkConfig
```

Comment:
If you're going to use CustomNetworkConfig below, you probably should keep the import